### PR TITLE
Newsletter-132:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

**Comment by jnewbery** [comment]
Yay! Very happy to see Japanese translations happening again :grin:

File: _posts/ja/newsletters/2021-01-20-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  エアギャップでnative segwitマルチシグのvaultを作成し管理する機能が追加されました。
+
+- **SparrowがBitcoin Coreへの接続をサポート:**
+  [Sparrow 0.9.10][sparrow 0.9.10]では、[Bitcoin Wallet Tracker v0.2.1][bwt 0.2.1]の
+  Java Native Interfaceバインディング機能を使って、Bitcoin Coreノードへの直接接続をサポートするようになりました。
+
+## リリースとリリース候補
+
+*人気のBitcoinインフラストラクチャプロジェクトの新しいリリースとリリース候補。
+新しいリリースにアップグレードするかリリース候補のテストを支援することを検討してください。*
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-01-20-newsletter.md:55]: s/するか/したり/

Comment (azuchi) [reply _posts/ja/newsletters/2021-01-20-newsletter.md]: Thanks! @AkioNak. I fixed.

**Comment by AkioNak** [comment]
I think it have been an easy readable sentences in Japanese. Thanks @azuchi !

**Comment by bitschmidty** [comment]
@azuchi thank you very much for this

@AkioNak thank you for the review

I have squashed the commits into one commit. Confirm this looks ok to you @azuchi , and I will merge.

**Comment by azuchi** [comment]
@bitschmidty thank you! I confirmed, its ok.

**Pull request merged**